gammapy/utils/scripts.py

```diff
@@ -2,7 +2,7 @@
 """Utils to create scripts and command-line tools"""
 import sys
 import logging
-import yaml
+import oyaml as yaml
```

Comment:
@QRemy - Can you use `pyyaml` for now?

Recently there was a lot of discussion which YAML package to use, and the conclusion was to use `pyyaml`, see:
https://github.com/gammapy/gammapy/blob/321cb278c6f1f507fb322468f70fa02c8d9d41d4/docs/development/pigs/pig-013.rst#alternatives

Do you absolutely need ordered dict for now? If you only read YAML and don't write YAML text and assert on it, then undefined order shouldn't matter for now, no?

Reply:
Ordered yaml files would be more human readable.
oyaml is based on pyyaml it just adds a  representer to enforce order when writing the file:
https://github.com/wimglenn/oyaml/blob/master/oyaml.py
if you don't want to import this module we can just copy the code to utils/scripts, no ?

Reply:
Suggest to try using `sort_keys=False` with pyyaml
```
>>> import yaml
>>> yaml.dump({2:2, 1:1}, sort_keys=False)
'2: 2\n1: 1\n'
```
Added very recently: https://github.com/yaml/pyyaml/blob/5986257f9fc978d4a61b6e0001df554f80e565cb/CHANGES#L33

If there's CI fails, I can help to fix them up by adjusting pyyaml versions.